### DISCUSSION TITLE
Very happy with how my MiniDexed synth turned out.

### DISCUSSION THREAD

**Discussion by randyandrade** [probonopd/MiniDexed#217] [Show and tell]
I just wanted to show off my (very rough) homemade MiniDexed Synthesizer.

I took the basic RasPi setup and put it in a box, included a headphones output, and a main output with balanced 1/4" TRS L&R connections, added a USB --> MIDI DIN interface, and adjustable backlight & contrast on the LCD display.

Please excuse the obvious rough spots where my printer was being grumpy and buggered up the black on the labels a bit.

![IMG_1743 (2)](https://user-images.githubusercontent.com/23284626/168106595-3c4c051a-9d4a-42d8-877a-5966d3f9ee82.png)
![IMG_1756 (2)](https://user-images.githubusercontent.com/23284626/168106599-4a1e0ad6-2ae2-47a1-9944-bc6fa6845475.png)
![IMG_1757 (2)](https://user-images.githubusercontent.com/23284626/168106605-7b0b5e45-775e-4e0d-b968-23dbe4392670.png)

**Comment by craigyjp** [comment]
How did you make the balanced outputs? Does your sound card provide balanced outputs?

You case puts mine to shame...

![IMG_12052022_194156_(1350_x_1080_pixel).jpg](https://user-images.githubusercontent.com/51026305/168126162-d5253d77-af6c-46e1-b151-8a8851603b62.jpg)

**Reply by randyandrade** [reply]
I got balanced outputs by using a passive DI (direct input) box internally to take an unbalanced signal, convert it to balanced, and then XLR --> TRS cable into the TRS Female-Female panel-mount coupler.

Something like this:
https://www.amazon.com/dp/B001032GK8?ref_=cm_sw_r_cp_ud_dp_5092NE6ZHMHPWWWH79AW

**Comment by craigyjp** [comment]
Ok, thanks for that, makes sense.
